src/main/java/blackjack/domain/participant/Players.java

```diff
@@ -0,0 +1,42 @@
+package blackjack.domain.participant;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Players {
+
+    private static final int MIN_PLAYER_SIZE = 1;
+    private static final int MAX_PLAYER_SIZE = 4;
+
+    private final List<Player> players;
+
+    Players(List<Player> players) {
+        this.players = players;
+    }
+
+    static public Players from(List<String> playerNames, List<Integer> bettings) {
+        validateSize(playerNames, bettings);
+
+        List<Player> players = IntStream.range(0, playerNames.size())
+                .mapToObj(i -> new Player(playerNames.get(i), new Betting(bettings.get(i))))
+                .collect(Collectors.toList());
+
+        return new Players(players);
+    }
+
+    public List<Player> getPlayers() {
+        return players;
+    }
+
+    private static void validateSize(List<String> players, List<Integer> bettings) {
+        if (players.size() < MIN_PLAYER_SIZE || players.size() > MAX_PLAYER_SIZE) {
```

Comment:
```suggestion
        if (players.isEmpty() || players.size() > MAX_PLAYER_SIZE) {
```
최소 플레이어 수가 1 보다 커진다면 상수를 추가하고 마크가 만들어주신 로직으로 작성해야겠지만, 지금 상황에서는 위와 같이 바꿔주는게 가독성면에서 더 좋은 것 같아요 :)

Reply:
아래 에러 메시지에서 재사용 하기위해 남겨두는 것도 좋다고 생각이 들어 남겨두었어요. 
하지만 말씀하신대로 플레이어가 없어도 될 경우가 거의 명백히 없기 때문에 메시지 보다 중요한 로직의 가독성을 높이는 것이 좋아보이네요!